src/main/resources/static/front/css/flowN/N02-menu.css

```
@@ -823,6 +823,115 @@
+    box-shadow: inset 0 0 0 4px var(--primary-500);
+}
+
+.n02__detail-opt.is-selected .n02__detail-opt-price {
+    color: var(--primary-600);
+}
+
+/* 음성으로 방금 선택된 옵션 펄스 */
+.n02__detail-opt.ai-pulse {
+    animation: n02OptPulse 1.2s ease;
+}
+@keyframes n02OptPulse {
```

Stylelint (17.11.1)

[error] Expected keyframe name "n02OptPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)